DEVELOPMENT.md

```diff
@@ -17,19 +17,19 @@ cd docker
 Build the base image
 
 ```bash
-docker build base -t base
+docker build -t base -f base/Dockerfile .
```

Comment:
The readme can be simplified to what it was before. no need to specify the file specifically if you're not passing context. I can revert it in a followup.